packages/react/src/components/asorted/Icon/Icons.stories.tsx

```diff
@@ -44,15 +61,32 @@ export default Story;
 const ListTemplate = (args: IconProps) => {
   const theme = useTheme();
   const color = args.color || theme.colors.palette.neutral.c100;
+  const [search, setSearch] = useState("");
+  const s = search.toLowerCase();
 
   return (
-    <div>
-      {iconNames.map((name) => (
-        <span title={name}>
-          <Icon key={name} name={name} weight={args.weight} size={args.size} color={color} />
-        </span>
-      ))}
-    </div>
+    <Container>
+      <SearchInput value={search} onChange={setSearch} />
+      <ScrollArea
+        gridTemplateColumns="repeat(auto-fill, minmax(100px, 1fr));"
+        gridTemplateRows="repeat(150px);"
+        gridGap={4}
+        mt={4}
+      >
+        {iconNames
+          .sort((a: string, b: string) => {
```

Comment:
✨ A cool thing to implement would be to highlight the icons that match the search string, as I think it's not super obvious that the search bar does not actually filter the icons.

Reply:
just did that :+1:

Reply:
![](https://media.giphy.com/media/Od0QRnzwRBYmDU3eEO/giphy-downsized.gif)